src/appservice/Appservice.ts

```diff
@@ -612,6 +649,15 @@ export class Appservice extends EventEmitter {
                 }
             }
 
+            if (this.registration["de.sorunome.msc2409.push_ephemeral"] && req.body["de.sorunome.msc2409.ephemeral"]) {
+                for (let event of req.body["de.sorunome.msc2409.ephemeral"]) {
+                    LogService.info("Appservice", `Processing ephemeral event of type ${event["type"]}`);
+                    event = await this.processEphemeralEvent(event);
+                    // These events aren't tied to rooms, so just emit them as `ephemeral_event`
+                    this.emit("ephemeral_event", event);
```

Comment:
This could potentially be `unstable_ephemeral_event`, WDYT?

Reply:
meh, enabling the flag using some non-standard namespace and bolded as **Experimental** is probably fine enough.